internal/backupcontroller/mariadbstrategy_controller.go

```
@@ -0,0 +1,814 @@
+	backupsv1alpha1 "github.com/cozystack/cozystack/api/backups/v1alpha1"
+	"github.com/cozystack/cozystack/internal/backupcontroller/mariadbapp"
+	"github.com/cozystack/cozystack/internal/backupcontroller/mariadbtypes"
+	"github.com/cozystack/cozystack/internal/template"
+)
+
+// ---------------------------------------------------------------------------
+// Constants
+// ---------------------------------------------------------------------------
+
+const (
+	mariadbFieldManager = "cozystack-mariadb-backup-driver"
```

golangci-lint (2.12.1)

[error] const mariadbFieldManager is unused

(unused)